### PR TITLE
Moving Toychain to Coq 8.11

### PR DESCRIPTION
**Pull request by ilyasergey**
* Minor fixes in the proof scripts for Coq 8.11
* Travis config file updated
* `README.md` updated
* `*.opam` descriptors updated

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -45,13 +45,19 @@ matrix:
     <<: *OPAM
   - env:
     - COQ_IMAGE=coqorg/coq:8.10
+    - CONTRIB_NAME=coq-toychain-node
+    - NJOBS=2
+    - SWITCH=4.07.1+flambda
+    <<: *OPAM
+  - env:
+    - COQ_IMAGE=coqorg/coq:8.11
```
Comment (palmskog) [inline .travis.yml]: If speed is an issue, we can use the MathComp image to save some time:
```suggestion
    - COQ_IMAGE=mathcomp/mathcomp:1.10.0-coq-8.11
```

File: .travis.yml [hunk 2]
```diff
@@ -45,13 +45,19 @@ matrix:
     <<: *OPAM
   - env:
     - COQ_IMAGE=coqorg/coq:8.10
```
Comment (palmskog) [inline .travis.yml]: Same as other comment.
```suggestion
    - COQ_IMAGE=mathcomp/mathcomp:1.10.0-coq-8.10
```

File: README.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Executable node:
 We recommend installing the Coq requirements via [OPAM](https://opam.ocaml.org/doc/Install.html):
 ```
 opam repo add coq-released https://coq.inria.fr/opam/released
-opam install coq-mathcomp-ssreflect coq-fcsl-pcm
+opam install coq coq-mathcomp-ssreflect coq-fcsl-pcm
```
Comment (palmskog) [inline README.md]: We may want to nudge the MathComp version here:
```suggestion
opam install coq coq-mathcomp-ssreflect.1.10.0 coq-fcsl-pcm
```

**Comment by dranov** [comment]
Thanks @palmskog!

**Pull request merged**